components/shared-image-gallery/image_gallery.tf

```
@@ -4,4 +4,18 @@ resource "azurerm_shared_image_gallery" "image_gallery" {
   location            = azurerm_resource_group.image_gallery_rg.location
 
   tags = module.ctags.common_tags
+}
+
+resource "azurerm_shared_image" "bastion-ubuntu" {
+  name                = "bastion-ubuntu"
+  gallery_name        = azurerm_shared_image_gallery.image_gallery.name
+  resource_group_name = azurerm_resource_group.image_gallery_rg.name
+  location            = azurerm_resource_group.image_gallery_rg.location
+  os_type             = "Linux"
+
+  identifier {
+    publisher = "Canonical"
```

shouldn't this be our image?

https://portal.azure.com/#@HMCTS.NET/resource/subscriptions/bf308a5c-0624-4334-8ff8-8dca9fd43783/resourceGroups/cnp-image-gallery-rg/providers/Microsoft.Compute/galleries/cnpimagegallery/images/jenkins-agent/overview

```suggestion
    publisher = "hmcts"
```